docs/index.md

```diff
@@ -2525,6 +2530,51 @@ object will read the local file instead.
       'timeseries/51922d0b37203f2a8c000010', connection);
 ```
 
+External Connectors
+-------------------
+
+BigML offers an `externalconnector` resource that can be used to connect to
+external data sources. The description of the API requirements to create an
+`ExternalConnector` can be found in the
+[API documentation](https://bigml.com/api/externalconnectors). The required
+information to create an external connector are parameters like the type
+of database manager, the host, user, password and table that we need to access.
+This must be provided as first argument to the `bigml.ExternalConnector`
+create method or can be set in environment variables:
```

Comment:
you could make this more explicit saying: or can be set in environment variables, in which case you pass the first argument as undefined.